verifier/builtins.go

```diff
@@ -0,0 +1,60 @@
+/*
+ * ZCrypto Copyright 2017 Regents of the University of Michigan
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not
+ * use this file except in compliance with the License. You may obtain a copy
+ * of the License at http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or
+ * implied. See the License for the specific language governing
+ * permissions and limitations under the License.
+ */
+
+package verifier
+
+import "github.com/zmap/zcrypto/x509"
+
+// Built-in verifiers representing common root stores.
+var (
+	// NSS is a Verifier mimicing the validation used in Firefox.
+	NSS Verifier
```

Comment:
Maybe put these in a map that is keyed by an enum? I'm not set on it, but if you wanted to do all of these at once, it would make it a bit easier